### PR TITLE
Added postCartItems method

### PR DESCRIPTION
**Pull request by GeluHai**
- Added postCartItems method
- Added test to addItemsToCart

### CODE REVIEW

File: src/main/java/api/domain/cart/items/AddItemToCart.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+package api.domain.cart.items;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AddItemToCart {
```
Comment (softvision-adrian-rosu) [inline src/main/java/api/domain/cart/items/AddItemToCart.java]: This class is redundant, please use Skus.java, all these fields are already in there. Objects should match the API naming, we should avoid creating objects for every operation we need in tests.

File: src/test/java/api/cart/PostCartTests.java [hunk 2]
```diff
@@ -15,4 +17,13 @@ public void createEmptyCart() {
...
         Assert.assertNotNull(postCartResponse.getId(), "id is null");
     }
+
+    @Test
+    public void addItemsToCart() {
+        Cart cart = new Cart();
+        AddItemToCart itemToAdd = CartUtils.addItemToCart("30262", "5");
+        Cart postCartResponse = cartApiWrapper.postEmptycart(cart);
+        Cart postItemToCartResponse = cartApiWrapper.postCartItems(itemToAdd, postCartResponse.getId());
+        Assert.assertNotNull(postItemToCartResponse.getId(), "id is null");
```
Comment (softvision-adrian-rosu) [inline src/test/java/api/cart/PostCartTests.java]: Please also add assert for status code, this check should be with highest priority.

**Review by softvision-adrian-rosu — CHANGES_REQUESTED** [review]

File: src/test/java/api/cart/PostCartTests.java (hunk 2 above)
Comment (GeluHai) [reply src/test/java/api/cart/PostCartTests.java]: done

File: src/main/java/api/domain/cart/items/AddItemToCart.java (hunk 1 above)
Comment (GeluHai) [reply src/main/java/api/domain/cart/items/AddItemToCart.java]: done

**Comment by GeluHai** [comment]
Added all the requested changes, please review.

File: src/main/java/api/domain/cart/items/Skus.java [hunk 3]
```diff
@@ -12,7 +12,7 @@
     @JsonProperty("product_id")
     private String productId;
     @JsonProperty("sku_id")
-    private Integer skuId;
+    private Integer sku_id;
```
Comment (softvision-adrian-rosu) [inline src/main/java/api/domain/cart/items/Skus.java:15]: @GeluHai it should work as private Integer skuId; you already have @JsonProperty set with the right naming. You only need to generate getters and setters

**Review by softvision-adrian-rosu — APPROVED** [review]

File: src/main/java/api/domain/cart/items/Skus.java (hunk 3 above)
Comment (GeluHai) [reply src/main/java/api/domain/cart/items/Skus.java:15]: If I use private Integer skuId it doesn't work, I tried

**Pull request merged**